### PR TITLE
add tinyply/2.3.2

### PR DESCRIPTION
**Pull request by SpaceIm**
Specify library name and version:  **tinyply/2.3.2**

- [x] I've read the [guidelines](https://github.com/conan-io/conan-center-index/blob/master/docs/how_to_add_packages.md) for contributing.
- [x] I've followed the [PEP8](https://www.python.org/dev/peps/pep-0008/) style guides for Python code in the recipes.
- [x] I've used the [latest](https://github.com/conan-io/conan/releases/latest) Conan client version.
- [x] I've tried at least one configuration locally with the
      [conan-center hook](https://github.com/conan-io/hooks.git) activated.

### CODE REVIEW

**Comment by SpaceIm** [comment]
MT/shared?

**Comment by madebr** [comment]
> MT/shared?

I think so. I think the allocator is not shared between exe and dll.
In that case, almost all (allocating) classes in the c++ standard libraries are unusable.

**Comment by SpaceIm** [comment]
While packaging this library, I'm wondering if header_ony libraries with macro implementation like `cgltf` or `stb` shouldn't have been packaged in another way (always compiled).

**Comment by uilianries** [comment]
I don't think so, because the package content is the same.

**Comment by intelligide** [comment]
> shouldn't have been packaged in another way

It will cause errors if the consumer defines the macro.

**Review by intelligide — APPROVED** [review]

**Review by SSE4 — APPROVED** [review]

File: recipes/tinyply/all/CMakeLists.txt [hunk 1]
```diff
@@ -0,0 +1,7 @@
+cmake_minimum_required(VERSION 3.4)
```
Comment (prince-chrismc) [inline recipes/tinyply/all/CMakeLists.txt:1]:
```suggestion
cmake_minimum_required(VERSION 3.1)
```

**Review by prince-chrismc — APPROVED** [review]

File: recipes/tinyply/all/CMakeLists.txt (hunk 1 above)
Comment (SpaceIm) [reply recipes/tinyply/all/CMakeLists.txt:1]: 3.4 due to https://github.com/ddiakopoulos/tinyply/blob/79a5c65965d193425dd33406a23c028a7aa29544/CMakeLists.txt#L19

**Pull request merged**